src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -219,27 +224,46 @@ export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDest
   }
 
   private initializeDragAndDrop(): void {
-    this.dragulaService
-      .drag
-      .takeUntil(this.ngUnsubscribe)
-      .subscribe(([, source]: Array<HTMLElement>) =>
-        source.classList.add('sky-repeater-item-dragging')
-      );
-
-    this.dragulaService
-      .dragend
-      .takeUntil(this.ngUnsubscribe)
-      .subscribe(([, source]: Array<HTMLElement>) => {
-        source.classList.remove('sky-repeater-item-dragging');
-        this.emitTags();
-      });
-
-    this.dragulaService.setOptions('sky-repeater', {
+    this.dragulaService.setOptions(this.dragulaGroupName, {
       moves: (el: HTMLElement, container: HTMLElement, handle: HTMLElement) => {
         const target = el.querySelector('.sky-repeater-item-grab-handle');
-        return target && target.contains(handle);
+        return (target && target.contains(handle));
       }
     });
+
+    // Reset the current dragula subscriptions.
+    this.dragulaUnsubscribe.next();
+    this.dragulaUnsubscribe.complete();
+    this.dragulaUnsubscribe = new Subject<void>();
```

Comment:
Could we reuse your `destroyDragAndDrop` method here?

Reply:
We can't because once the dragula group is initialized and destroyed (see `this.dragulaService.destroy`), it can't be re-initialized afterwards. We need to be able to reinitialize the dragula group if `reorderable` changes.